Modules/Filtering/FastMarching/include/itkFastMarchingImageToNodePairContainerAdaptor.h

```diff
@@ -53,7 +53,7 @@ class ITK_TEMPLATE_EXPORT FastMarchingImageToNodePairContainerAdaptor : public O
   ITK_DISALLOW_COPY_AND_MOVE(FastMarchingImageToNodePairContainerAdaptor);
 
   using Self = FastMarchingImageToNodePairContainerAdaptor;
-  using Superclass = LightObject;
+  using Superclass = Object;
```

Comment:
I guess this is the right inheritance. Let me know otherwise (i.e. if `LightObject` should be the parent class instead in https://github.com/InsightSoftwareConsortium/ITK/pull/3382/files#diff-15a472539026d9182f07806b46297e6a0fe90ab0012067872bad81ec3c62a638R50).